frontend/modules/dialog/dialog.redux.ts

```diff
@@ -18,7 +18,9 @@
 import { createActions, createReducer } from 'reduxsauce';
 import { get, omit } from 'lodash';
 import { ScreenshotDialog } from '../../routes/components/screenshotDialog/screenshotDialog.component';
-import { ErrorDialog, ConfirmDialog } from '../../routes/components/dialogContainer/components';
+import {
+	ErrorDialog, ConfirmDialog, DisconnectedDialog, RevisionsDialog, SimpleErrorDialog, NewUpdateDialogDialog
```

Comment:
I think the number of dialogs will increase. Could you change that import to `import * as Dialogs ...`?